src/App.js

```diff
@@ -1,20 +1,39 @@
 import "./App.css";
 import {Route,Routes} from "react-router-dom"
-import { Navbar } from "./Components/Navigation/Navbar";
-import { Sidebar } from "./Components/Sidebar/Sidebar";
-import { VideoCard } from "./Components/VideoCards/VideoCard";
 import MockmanEs from "mockman-js";
+import { Login } from "./Components";
+import { Signup } from "./Components/LoginSignUp/SignUp";
+import {VideoCard} from "./Components/VideoCards/VideoCard"
+import {NavContainer} from "./Components/NavRoutes/NavContainer"
```

Comment:
add index.js file in components folder, export all component in that index file and then import it here from index file

Reply:
ok i will do